src/test/resources/munit/orders-model.raml

```diff
@@ -0,0 +1,21 @@
+#%RAML 1.0 Library
```

Comment:
I would create separate ramls for each test-suite, so each resource is focalized on an specific use case. I know this is a PAIN in the a** but worth it long term. Imagine that you need to create a new test suite for the feature X and you need to reuse this RAML to add the resources that the feature X requires, then this is going to get bigger and there is no way to distinct easily what this raml aims to represent. Following this thought think then when on top of X you need to add tests for the feature Y,Z,... this will be a P.I.T.A.